mne_bids/mne_bids.py

```diff
@@ -520,9 +520,6 @@ def _sidecar_json(raw, task, manufacturer, fname, kind, overwrite=False,
         append_kind_json = ch_info_json_eeg
     elif kind == 'ieeg':
         append_kind_json = ch_info_json_ieeg
-    else:
```

Comment:
I removed this because it is impossible to reach this code. The function that returns the `kind` never returns anything other than the above 3 values (if it doesn't like the data it receives it raises an error...)